atlasdb-workload-server-antithesis/var/compact.sh

```diff
@@ -0,0 +1,18 @@
+##
+# (c) Copyright 2023 Palantir Technologies Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#!/bin/bash
```

Comment:
For scripts this typically _has_ to be the first line (even above any license stuff), otherwise the default shell will get executed which is not guaranteed to be bash. We're just running one command so not reliant on bash specifically, but it could be misleading